.github/workflows/frontend.yaml

```diff
@@ -24,6 +24,26 @@ jobs:
       - name: Install dependencies
         run: yarn --frozen-lockfile
 
+      - name: Get latest Graph Proxy Schema run id
+        run: |
+          GRAPH_WORKFLOW_ID=$(gh api "/repos/DiamondLightSource/workflows/actions/workflows/119673956/runs?status=success" --jq ".workflow_runs[0].id")
```

Comment:
I don't think this guarantees that we get the workflow from this commit, just the latest instance of this workflow